packages/blip/app/components/patient-data.js

```diff
@@ -643,7 +643,7 @@ class PatientDataPage extends React.Component {
     // Default one week data period for dashboard (now() - 7 days)
     this.setState({
       epochLocation: new Date().valueOf(),
-      msRange: TimeService.MS_IN_DAY * 7
+      msRange: TimeService.MS_IN_DAY * 14
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏This could be put in a const at the top of the file to avoid this magic number (also, the comment needs to be updated)